recipe/build.sh

```diff
@@ -1,16 +1,35 @@
 #!/usr/bin/env sh
+
+set -euxo pipefail
+
 # Get an updated config.sub and config.guess
 cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./build-aux
 cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./libcharset/build-aux
-set -ex
+
+# this bit of code will refresh the flags.h header
+# for now we have hard coded the changes as a patch.
+# if [[ "${target_platform}" == osx-* ]]; then
+#     mv lib/flags.h lib/flags.h.bak
+#     ${CC_FOR_BUILD} ${CFLAGS} lib/genflags.c -o genflags
+#     ./genflags > lib/flags.h
+#     rm -f genflags
+#     # Debugging: Show generated diff
+#     diff -u lib/flags.h.bak lib/flags.h || true
+#     # Check that UTF-8.MAC is included
+#     grep utf8mac lib/flags.h
+# fi
 
 ./configure --prefix=${PREFIX}  \
             --host=${HOST}      \
             --build=${BUILD}    \
             --enable-static     \
             --disable-rpath
 
-make -j${CPU_COUNT} ${VERBOSE_AT}
-if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
+if [[ "${target_platform}" == osx-* ]]; then
+    make -f Makefile.devel CC="${CC}" CFLAGS="${CFLAGS}"
```

Comment:
The alias generation in `Makefile.devel` runs `gperf` -- we may want to add that to the build requirements for completeness' sake.

Reply:
(On `unix` at least.)